drv/ignition-api/src/lib.rs

```diff
@@ -8,16 +8,16 @@
 
 use bitfield::bitfield;
 use derive_idol_err::IdolError;
-use derive_more::From;
+use derive_more::{Display, From, LowerHex, UpperHex};
 use num_derive::{FromPrimitive, ToPrimitive};
 use num_traits::FromPrimitive;
 use zerocopy::{AsBytes, FromBytes};
 
-// The `presence_summary` vector (see `ignition-server`) is implicitly capped at 40 bits by (the RTL of
-// the) mainboard controller. This constant is used to conservatively allocate
-// an array type which can contain the port state for all ports. The actual
-// number of pors configured in the system can be learned through the
-// `port_count()` function below.
+// The `presence_summary` vector (see `ignition-server`) is implicitly capped at
+// 40 bits by (the RTL of the) mainboard controller. This constant is used to
+// conservatively allocate an array type which can contain the port state for
+// all ports. The actual number of pors configured in the system can be learned
+// through the `port_count()` function below.
 pub const PORT_MAX: usize = 40;
```

Comment:
I'd vote for this to be a `u8`: in different places, we use it as both a `u8` and a `usize`; making the constant a `u8` is the safer choice.

Reply:
Yes, this is the right thing to do. The places where it's used as `usize` are pretty minimal.